proposals/mips/mips.json

```diff
@@ -1,4 +1,11 @@
 [
+    {
+        "envPath": "",
+        "governor": "MultichainGovernor",
+        "id": 86,
```

Comment:
please leave this as 0, it should only be included once the proposal is on chain